wave_vision/tests/matcher_tests/brute_force_tests.cpp

```diff
@@ -0,0 +1,137 @@
+// C++ Headers
+#include <algorithm>
+
+// Libwave Headers
+#include "wave/wave_test.hpp"
+#include "wave/vision/detector/fast_detector.hpp"
+#include "wave/vision/descriptor/brisk_descriptor.hpp"
+#include "wave/vision/matcher/brute_force_matcher.hpp"
+
+namespace wave {
+
+const auto TEST_CONFIG = "tests/config/matcher/brute_force_matcher.yaml";
+const auto TEST_IMAGE_1 = "tests/data/image_center.png";
+const auto TEST_IMAGE_2 = "tests/data/image_right.png";
+
+// Test fixture to load image data
+class BFTest : public testing::Test {
+ protected:
+    BFTest() {
+        initDetector();
+        initDescriptor();
+        initMatcher();
+        this->image_1 = cv::imread(TEST_IMAGE_1, cv::IMREAD_COLOR);
+        this->image_2 = cv::imread(TEST_IMAGE_2, cv::IMREAD_COLOR);
+        this->keypoints_1 = this->fast.detectFeatures(this->image_1);
+        this->keypoints_2 = this->fast.detectFeatures(this->image_2);
```

Comment:
I mean - this is just to make sure that the matches actually are being computed and that they make sense. Really testing this is just making sure OpenCV's modules work. Just want to make sure that I am in fact able to see proper matches between objects.

I guess I could check if the size of matches is greater than zero as well, but I feel like the visual check here is more representative. What do you suggest @leokoppel?